tests/waku_rln_relay/test_wakunode_rln_relay.nim

```diff
@@ -1,23 +1,18 @@
 {.used.}
 
 import
-  std/[sequtils, tempfiles],
+  std/[tempfiles, sequtils],
```

Comment:
isn't s before t in the alphabet?:D

Reply:
err, my bad - that change shouldn't be in the diff